content/en/data_streams/technologies/azure_service_bus.md

```diff
@@ -0,0 +1,32 @@
+---
+title: Azure Service Bus for Data Streams Monitoring
+---
+
+### Prerequisites
+
+* [Datadog Agent v7.34.0 or later][1]
+
+{{% data_streams/monitoring-azure-service-bus %}}
+
+### Support for Azure Service Bus in Data Streams Monitoring
+
+<table>
+  <thead>
+    <tr>
+      <th>Language</th>
+      <th>Library</th>
+      <th>Minimal tracer version</th>
+      <th>Recommended tracer version</th>
+    </tr>
+  </thead>
+  <tbody>
+    <tr>
+      <td><a href="/data_streams/dotnet">.NET</a></td>
+      <td><a href="https://www.nuget.org/packages/Azure.Messaging.ServiceBus">Azure.Messaging.ServiceBus</a></td>
+      <td>2.53.0 </td>
+      <td>2.53.0 or later </td>
+    </tr>
+  </tbody>
+</table>
```

Comment:
decided to use html table styling for these for consistency. Some of the tech tables have cells that span multiple columns (kafka's Go language has multiple libraries for ex.)